CMSgov/easi-app
# Easi 3248/request edits email

# EASI-3248

## Changes and Description

- Create Request Edits Email
    - Create System Intake Edits Requested Email
       - New struct on email client to separate calls to IT gov v2 emails


## How to test this change
Use postman to do the following ( or use the front end)
1. Create a system intake and submit it
2. Request edits to the system intake
3. Confirm that you receive an email in mail catcher
4. Send another edits requested email that does not include additionalInfo. 
5. Confirm that the additional information section is left out of the email


Note, using the listAllRecipients method does introduce the ability to send to the IT Governance and IT Investment inbox. If desired we could remove the ability to send to the It investment inbox, but it would be ignoring the properties provided on the `EmailNotificationRecipients` type.
## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
